backend/vendor/assets/stylesheets/jquery.jstree/themes/spree/style.scss

```diff
@@ -0,0 +1,216 @@
+  }
+
+  .jstree-open > ins:before { content: '\e114' }
+
+  .jstree-closed > ins:before { content: '\e080' }
+
+  .jstree-leaf > ins {
+    background-position: -36px 0;
+  }
+
+  .jstree-locked a {
+    color: silver;
+    cursor: default;
+  }
+
+  a {
+    line-height: 38px;
+    height: 38px;
+    padding: 0;
+    color: #555;
+    cursor: context-menu;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    user-select: none;
+
+    &.jstree-search {
+      color: aqua;
+    }
+
+    &.jstree-loading .jstree-icon {
+      background: image-url("jquery.jstree/themes/spree/throbber.gif") center center no-repeat !important;
+
+      &:before { content: '' }
+    }
+
+    .jstree-icon {
+      vertical-align: middle;
+      font-family: 'Glyphicons Halflings';
+      line-height: 20px;
+      width: 20px;
+      height: 24px;
+      text-align: center;
+      margin-right: 8px;
+      pointer-events: none;
+
+      &:before {
+        content: "\e118";
+        top: 1px;
+        padding-left: 0;
+      }
+    }
+  }
+
+  .jstree-rename-input {
+    height: 38px !important;
+    line-height: 38px !important;
+    left: 46px !important;
+    padding: 0 5px !important;
+  }
+
+  .jstree-hovered {
+    background: #e7f4f9;
+    border: 1px solid #d8f0fa;
+    padding: 0 3px 0 1px;
+    text-shadow: 1px 1px 1px silver;
+  }
+
+  .jstree-clicked {
+    background: #beebff;
+    border: 1px solid #99defd;
+    padding: 0 3px 0 1px;
+  }
+
+  .jstree-no-dots li,
+  .jstree-no-dots .jstree-leaf > ins {
+    background: transparent;
+  }
+
+  .jstree-no-dots .jstree-open > ins {
+    background-position: -18px 0;
+  }
+
+  .jstree-no-dots .jstree-closed > ins {
+    background-position: 0 0;
+  }
+
+  .jstree-no-icons a .jstree-icon {
+    display: none;
+  }
+
+  .jstree-search {
+    font-style: italic;
+  }
+
+  .jstree-no-icons .jstree-checkbox {
+    display: inline-block;
+  }
+
+  .jstree-no-checkboxes .jstree-checkbox {
+    display: none !important;
+  }
+
+  .jstree-checked > a > .jstree-checkbox {
+    background-position: -38px -19px;
+  }
+
+  .jstree-unchecked > a > .jstree-checkbox {
+    background-position: -2px -19px;
+  }
+
+  .jstree-undetermined > a > .jstree-checkbox {
+    background-position: -20px -19px;
+  }
+
+  .jstree-checked > a > .checkbox:hover {
+    background-position: -38px -37px;
+  }
+
+  .jstree-unchecked > a > .jstree-checkbox:hover {
+    background-position: -2px -37px;
+  }
+
+  .jstree-undetermined > a > .jstree-checkbox:hover {
+    background-position: -20px -37px;
+  }
+}
+
+#vakata-dragged.jstree-spree ins {
+  background: transparent !important;
+}
+
+#vakata-contextmenu.jstree-spree-context {
+  background: #fbfbfb;
+  border: 1px solid #eee;
+  margin-top: 15px;
+  margin-left: -4px;
+  -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+  -moz-box-shadow:    0 6px 12px rgba(0, 0, 0, 0.175);
+  box-shadow:         0 6px 12px rgba(0, 0, 0, 0.175);
+
+  li {
+
+    ins { display: none }
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Declaration should be terminated by a semicolon